src/main/java/uk/gov/hmcts/reform/civil/service/GeneralAppsDeadlinesCalculator.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.civil.service;
+
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+
+@Service
+public class GeneralAppsDeadlinesCalculator extends DeadlinesCalculator {
```

Comment:
Do we need 2 separate classes? In civil-service, there are 2 classes as the default calculator (created for claims) does not produce date as per our requirements so we had to extend it to override a method for our purpose. We can have a single calculator here

Reply:
Done. Cheers!